### ISSUE TITLE
Add on hover to entity type icons on results pages, entity pages, advanced search (any place icons are used) (from 1607)

### ISSUE THREAD

**Issue by gigamorph** [project-lux/lux-frontend#22]
**Problem description:**
The icons displayed alongside the results and in the entity page headers do not have visible labels displayed.

**Expected Behavior:** 
When hovering over an icon, the user should be shown text that describes the icon. There should be no hover delay. The label will be visible/invisible to the user immediately when their cursor enters/leaves the icon area.  This should be implemented wherever icons are used in the UI. 

**Requirements**
- Add hover on the icons on result pages, entity pages, and advanced search results. 
- Display the correct icon label in the hover text. 

**Needed for promotion**
- [x] Labels <-- listed below but need to be updated and reviewed
- [x] Mockup
- [x] Final list of icon designs and labels in this top block
- [x] Links/Directions for UAT Testing each

<img width="301" alt="image" src="https://github.com/project-lux/lux-frontend/assets/160149818/8ecdde29-56d3-4737-8b5d-36d0e9bfc66f">

**UAT**
Current Labels and link for UAT
1. [Scores and Notation](https://lux-front-tst.collections.yale.edu/view/results/objects?q=%7B%22text%22%3A%22video%22%2C%22_lang%22%3A%22en%22%7D&sq=video&ip=1&if=%7B%22AND%22%3A%5B%7B%22hasDigitalImage%22%3A0%7D%2C%7B%22classification%22%3A%7B%22id%22%3A%22https%3A%2F%2Flux-front-tst.collections.yale.edu%2Fdata%2Fconcept%2F553abb46-340c-46b0-9a19-aa9eb912db78%22%7D%7D%5D%7D&facetRequest=true)
2. [Photographs](https://lux-front-tst.collections.yale.edu/view/results/objects?q=%7B%22text%22%3A%22robe%22%2C%22_lang%22%3A%22en%22%7D&sq=robe&ip=1&if=%7B%22AND%22%3A%5B%7B%22classification%22%3A%7B%22id%22%3A%22https%3A%2F%2Flux-front-tst.collections.yale.edu%2Fdata%2Fconcept%2Fd9a18306-8983-4bef-b15d-aca7641b818c%22%7D%7D%5D%7D&facetRequest=true)
3. [Furniture](https://lux-front-tst.collections.yale.edu/view/results/objects?q=%7B"text"%3A"robe"%2C"_lang"%3A"en"%7D&sq=robe&ip=1&if=%7B"AND"%3A%5B%7B"classification"%3A%7B"id"%3A"https%3A%2F%2Flux-front-tst.collections.yale.edu%2Fdata%2Fconcept%2Ff2f3bfa3-97e2-46d6-a01c-55969e08991e"%7D%7D%5D%7D&facetRequest=true)
4. [Journals & Newspapers](https://lux-front-tst.collections.yale.edu/view/results/objects?q=%7B%22AND%22%3A%5B%7B%22text%22%3A%22washington%22%2C%22_lang%22%3A%22en%22%7D%2C%7B%22text%22%3A%22post%22%2C%22_lang%22%3A%22en%22%7D%5D%7D&sq=washington+post&ip=1&if=%7B%22AND%22%3A%5B%7B%22classification%22%3A%7B%22id%22%3A%22https%3A%2F%2Flux-front-tst.collections.yale.edu%2Fdata%2Fconcept%2F5ef5aa2a-669c-449e-8a32-5582c5f96514%22%7D%7D%5D%7D&facetRequest=true)
5.  [Clothing](https://lux-front-tst.collections.yale.edu/view/results/objects?q=%7B%22text%22%3A%22garment%22%2C%22_lang%22%3A%22en%22%7D&sq=garment&ip=1&if=%7B%22AND%22%3A%5B%7B%22hasDigitalImage%22%3A0%7D%2C%7B%22memberOf%22%3A%7B%22curatedBy%22%3A%7B%22OR%22%3A%5B%7B%22memberOf%22%3A%7B%22id%22%3A%22https%3A%2F%2Flux-front-tst.collections.yale.edu%2Fdata%2Fgroup%2F41310ca5-8137-45fe-ac2c-a6a04e2235f1%22%7D%7D%2C%7B%22id%22%3A%22https%3A%2F%2Flux-front-tst.collections.yale.edu%2Fdata%2Fgroup%2F41310ca5-8137-45fe-ac2c-a6a04e2235f1%22%7D%5D%7D%7D%7D%5D%7D&facetRequest=true)
6. [Numismatics](https://lux-front-tst.collections.yale.edu/view/results/objects?q=%7B%22text%22%3A%22coin%22%2C%22_lang%22%3A%22en%22%7D&sq=coin&ip=1&if=%7B%22AND%22%3A%5B%7B%22hasDigitalImage%22%3A0%7D%2C%7B%22classification%22%3A%7B%22id%22%3A%22https%3A%2F%2Flux-front-tst.collections.yale.edu%2Fdata%2Fconcept%2F7d74b40b-467d-45c0-876a-9a7a66b2b457%22%7D%7D%5D%7D&facetRequest=true)
7.  [Cartography](https://lux-front-tst.collections.yale.edu/view/results/objects/?q=%7B%22text%22%3A%22map%22%2C%22_lang%22%3A%22en%22%7D&sq=map&ip=1&if=%7B%22AND%22%3A%5B%7B%22classification%22%3A%7B%22id%22%3A%22https%3A%2F%2Flux-front-tst.collections.yale.edu%2Fdata%2Fconcept%2F6e901924-9ff3-4056-a1ce-71dce5424b33%22%7D%7D%5D%7D&facetRequest=true)
8. [Video](https://lux-front-tst.collections.yale.edu/view/results/objects?q=%7B%22text%22%3A%22video%22%2C%22_lang%22%3A%22en%22%7D&sq=video&ip=1&if=%7B%22AND%22%3A%5B%7B%22hasDigitalImage%22%3A0%7D%2C%7B%22classification%22%3A%7B%22id%22%3A%22https%3A%2F%2Flux-front-tst.collections.yale.edu%2Fdata%2Fconcept%2Facf2cace-57e0-4237-b9f9-6b6b5dc4c51c%22%7D%7D%5D%7D&facetRequest=true)
9. [Audio](https://lux-front-tst.collections.yale.edu/view/results/objects?q=%7B%22text%22%3A%22box%22%2C%22_lang%22%3A%22en%22%7D&sq=box&ip=1&if=%7B%22AND%22%3A%5B%7B%22hasDigitalImage%22%3A0%7D%2C%7B%22classification%22%3A%7B%22id%22%3A%22https%3A%2F%2Flux-front-tst.collections.yale.edu%2Fdata%2Fconcept%2F912de2b2-e751-4d25-9290-7e67a45ec34c%22%7D%7D%5D%7D&facetRequest=true)
10.  [Software & Electronic Media](https://lux-front-tst.collections.yale.edu/view/results/objects?q=%7B%22AND%22%3A%5B%7B%22text%22%3A%22software%22%2C%22_lang%22%3A%22en%22%7D%2C%7B%22text%22%3A%22electronic%22%2C%22_lang%22%3A%22en%22%7D%2C%7B%22text%22%3A%22media%22%2C%22_lang%22%3A%22en%22%7D%5D%7D&sq=software+and+electronic+media&ip=1&if=%7B%22AND%22%3A%5B%7B%22classification%22%3A%7B%22id%22%3A%22https%3A%2F%2Flux-front-tst.collections.yale.edu%2Fdata%2Fconcept%2F9eae32b4-1c3f-4323-84b1-e58ac8007a62%22%7D%7D%2C%7B%22hasDigitalImage%22%3A0%7D%5D%7D&facetRequest=true)
11. [Visual Works](https://lux-front-tst.collections.yale.edu/view/results/works?q=%7B%22text%22%3A%22o%27keeffe%22%2C%22_lang%22%3A%22en%22%7D&sq=o%27keeffe&ip=1&if=%7B%22AND%22%3A%5B%7B%22classification%22%3A%7B%22id%22%3A%22https%3A%2F%2Flux-front-tst.collections.yale.edu%2Fdata%2Fconcept%2Ff205dced-45a6-46d4-a4c5-efec14705c55%22%7D%7D%5D%7D&facetRequest=true&wp=1&wf=%7B%22AND%22%3A%5B%7B%22hasDigitalImage%22%3A0%7D%2C%7B%22classification%22%3A%7B%22id%22%3A%22https%3A%2F%2Flux-front-tst.collections.yale.edu%2Fdata%2Fconcept%2Ff205dced-45a6-46d4-a4c5-efec14705c55%22%7D%7D%5D%7D)
12. [Textual Works](https://lux-front-tst.collections.yale.edu/view/results/works?q=%7B%22text%22%3A%22o%27keeffe%22%2C%22_lang%22%3A%22en%22%7D&sq=o%27keeffe&ip=1&if=%7B%22AND%22%3A%5B%7B%22classification%22%3A%7B%22id%22%3A%22https%3A%2F%2Flux-front-tst.collections.yale.edu%2Fdata%2Fconcept%2Ff205dced-45a6-46d4-a4c5-efec14705c55%22%7D%7D%5D%7D&facetRequest=true&wp=1&wf=%7B%22AND%22%3A%5B%7B%22hasDigitalImage%22%3A0%7D%2C%7B%22recordType%22%3A%22LinguisticObject%22%7D%5D%7D)
13. [Objects](https://lux-front-tst.collections.yale.edu/view/results/objects?q=%7B%22AND%22%3A%5B%7B%22text%22%3A%22cultural%22%2C%22_lang%22%3A%22en%22%7D%2C%7B%22text%22%3A%22artifacts%22%2C%22_lang%22%3A%22en%22%7D%5D%7D&sq=cultural+artifacts&ip=1&if=%7B%22AND%22%3A%5B%7B%22classification%22%3A%7B%22id%22%3A%22https%3A%2F%2Flux-front-tst.collections.yale.edu%2Fdata%2Fconcept%2F8eede6bc-e29e-4bc7-a70c-dabdfbcdcf25%22%7D%7D%2C%7B%22hasDigitalImage%22%3A0%7D%5D%7D&facetRequest=true)
14. [Specimens](https://lux-front-tst.collections.yale.edu/view/results/objects?q=%7B%22text%22%3A%22specimen%22%2C%22_lang%22%3A%22en%22%7D&sq=specimen)
15. [Collections](https://lux-front-tst.collections.yale.edu/view/results/works?q=%7B%22text%22%3A%22o%27keeffe%22%2C%22_lang%22%3A%22en%22%7D&sq=o%27keeffe&ip=1&if=%7B%22AND%22%3A%5B%7B%22classification%22%3A%7B%22id%22%3A%22https%3A%2F%2Flux-front-tst.collections.yale.edu%2Fdata%2Fconcept%2Ff205dced-45a6-46d4-a4c5-efec14705c55%22%7D%7D%5D%7D&facetRequest=true&wp=1&wf=%7B%22AND%22%3A%5B%7B%22hasDigitalImage%22%3A0%7D%5D%7D)

**Dependencies/Blocks**
- Blocked By: https://github.com/project-lux/lux-frontend/issues/40

**Related Issues**
- https://www.bugherd.com/projects/284041/tasks/1592
- https://www.bugherd.com/projects/284041/tasks/1426

**Mockup/Wireframes:**
<img width="330" alt="image" src="https://github.com/project-lux/lux-frontend/assets/160149818/243b2497-ebee-412b-848c-8a7a0434caa4">

**Comment by roamye** [comment]
This is no longer blocked. Moving this to backlog and assigning @kamerynB. 

note to self: update links and reformat issue.

**Comment by roamye** [comment]
inconsistent with:
- photography: listed as "photographs" 
- numismatics: spelled incorrectly as "numistatics" 

things to note: 
Under works, if a work is a digital work, it will show "digital work" when hovering over it rather than (example) 
Under works, textual object icons also display "linguistic object". [example](https://lux-data-dev.collections.yale.edu/view/results/works?q=%7B%22text%22%3A%22o%27keeffe%22%2C%22_lang%22%3A%22en%22%7D&sq=o%27keeffe&ip=1&if=%7B%22AND%22%3A%5B%7B%22classification%22%3A%7B%22id%22%3A%22https%3A%2F%2Flux-front-tst.collections.yale.edu%2Fdata%2Fconcept%2Ff205dced-45a6-46d4-a4c5-efec14705c55%22%7D%7D%5D%7D&facetRequest=true&wp=1&wf=%7B%22AND%22%3A%5B%7B%22hasDigitalImage%22%3A0%7D%2C%7B%22recordType%22%3A%22LinguisticObject%22%7D%5D%7D)
Set icon will appear as 'set and collection" if applicable. I think this is good that it can differentiate it, but wanted to note it. 

Unsure if these are issues.

**Comment by kkdavis14** [comment]
I think we should swap the "linguistic object" hover over to "textual work" since that's what the UI uses generally.

**Comment by roamye** [comment]
uat 10/24: 
hot fixes: 
- Photography should be changed to photograph.
- numismatics to be spelled correctly. 

Quality control is needed for all the labels. 
- should they be lower case or capitalized? A: can be. 
- on tomorrows agenda. 
- [teams thread on conversation](https://teams.microsoft.com/l/message/19:351e0281aaa144d0812b960d216958c6@thread.tacv2/1729791011678?tenantId=dd8cbebb-2139-4df8-b411-4e3e87abeb5c&groupId=4620eac6-0955-4c69-93e8-b493b21f0df6&parentMessageId=1729791011678&teamName=ML%2F%20Cultural%20Heritage%20Support&channelName=1%20-%20LUX_ML&createdTime=1729791011678)

Next steps: 
- deploy to prod, but will need another ticket for re-labeling/analysis.

**Comment by clarkepeterf** [comment]
This is not in PRD, so I will deploy to lower envs

**Comment by roamye** [comment]
Approved by UAT, but as mentioned in the comment above, this will need another ticket for re-labeling.

**Comment by roamye** [comment]
Looks good in prod. closing
